resources/public/api/conf/1.0/schemas/businesses/create_response.json

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Create a Test Business",
+  "type": "object",
+  "properties": {
+    "businessId": {
+      "type": "string",
+      "description": "A unique identifier for the business income source.",
+      "pattern": "^X[A-Z0-9]{1}IS[0-9]{11}$"
+    },
+    "links": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A list of endpoint links that indicate possible actions related to the current resource.",
+        "properties": {
+          "href": {
+            "description": "The relative url of the endpoint.",
+            "type": "string",
+            "example": "https://test-api.service.hmrc.gov.uk/individuals/self-assessment-test-support/business/ET009363D"
+          },
+          "method": {
+            "description": "The HTTP method type for the endpoint.",
+            "type": "string",
+            "enum": [
+              "GET",
+              "DELETE"
+            ]
+          },
+          "rel": {
+            "description": "A label for the endpoint, which describes how it is related to the current resource\nThe rel will be self where the action is retrieval of the same resource.",
+            "type": "string",
+            "enum": [
+              "delete-business",
+              "list-all-businesses",
+              "retrieve-business-details"
```

Are you missing "self"?
```suggestion
              "self",
              "delete-business"
```

or (if you apply the other, optional suggestion)

```suggestion
              "self",
              "list-all-businesses",
              "delete-business"
```